2025-04-14 14:39:33,937 - cephci - log:161 - DEBUG - Completed log configuration
2025-04-14 14:39:33,939 - cephci - run:722 - INFO - Running test cephfs_nfs.2_node_nfs.py
2025-04-14 14:39:33,940 - cephci - 2_node_nfs:44 - INFO - Running cephfs CEPH-83574028 test case
2025-04-14 14:39:33,941 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251
2025-04-14 14:39:34,945 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 14:39:35,504 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:35,504 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 14:39:35,504 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:36,379 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 1:03:42 ago on Mon Apr 14 13:35:33 2025.
2025-04-14 14:39:36,610 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:36,611 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 14:39:36,613 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 14:39:36,614 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 14:39:36,615 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 14:39:36,616 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 14:39:36,617 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 14:39:36,619 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:36,620 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 14:39:36,621 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 14:39:36,714 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 14:39:36,715 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 14:39:36,716 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 14:39:36,833 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.251 took 2.891452 seconds
2025-04-14 14:39:36,875 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.251
2025-04-14 14:39:37,878 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 14:39:37,879 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 14:39:37,879 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 14:39:37,879 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 14:39:37,879 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 14:39:37,879 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 14:39:37,879 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.251 took 1.004307 seconds
2025-04-14 14:39:37,881 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.251
2025-04-14 14:39:38,884 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.251 took 1.003364 seconds
2025-04-14 14:39:38,886 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.251
2025-04-14 14:39:39,890 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.251 took 1.003078 seconds
2025-04-14 14:39:39,892 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3
2025-04-14 14:39:40,896 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 14:39:41,381 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:41,381 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 14:39:41,381 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:42,227 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 0:43:15 ago on Mon Apr 14 13:56:06 2025.
2025-04-14 14:39:42,461 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:42,462 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 14:39:42,463 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 14:39:42,464 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 14:39:42,465 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 14:39:42,467 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 14:39:42,468 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 14:39:42,469 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:42,470 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 14:39:42,471 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 14:39:42,564 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 14:39:42,565 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 14:39:42,565 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 14:39:43,676 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.3 took 3.784031 seconds
2025-04-14 14:39:43,680 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.3
2025-04-14 14:39:44,684 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 14:39:44,684 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 14:39:44,684 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 14:39:44,685 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 14:39:44,685 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 14:39:44,685 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 14:39:44,685 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.3 took 1.004107 seconds
2025-04-14 14:39:44,686 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.3
2025-04-14 14:39:45,690 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.3 took 1.003301 seconds
2025-04-14 14:39:45,691 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.3
2025-04-14 14:39:46,695 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.3 took 1.003756 seconds
2025-04-14 14:39:46,698 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202
2025-04-14 14:39:47,705 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 14:39:48,310 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:48,310 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 14:39:48,310 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:49,109 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 1:00:10 ago on Mon Apr 14 13:39:18 2025.
2025-04-14 14:39:49,342 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:49,342 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 14:39:49,343 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 14:39:49,345 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 14:39:49,346 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 14:39:49,347 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 14:39:49,348 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 14:39:49,349 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:49,350 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 14:39:49,351 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 14:39:49,446 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 14:39:49,447 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 14:39:49,447 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 14:39:49,845 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.202 took 3.146437 seconds
2025-04-14 14:39:49,887 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.202
2025-04-14 14:39:50,892 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 14:39:50,892 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 14:39:50,892 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 14:39:50,892 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 14:39:50,893 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 14:39:50,893 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 14:39:50,893 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.202 took 1.006022 seconds
2025-04-14 14:39:50,895 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.202
2025-04-14 14:39:51,899 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.202 took 1.003515 seconds
2025-04-14 14:39:51,901 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.202
2025-04-14 14:39:52,905 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.202 took 1.003716 seconds
2025-04-14 14:39:52,908 - cephci - ceph:1576 - INFO - Execute yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142
2025-04-14 14:39:53,919 - cephci - ceph:1186 - DEBUG - Updating Subscription Management repositories.
2025-04-14 14:39:54,422 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:54,422 - cephci - ceph:1186 - DEBUG - This system has release set to 9.5 and it receives updates only for this release.
2025-04-14 14:39:54,422 - cephci - ceph:1186 - DEBUG - 
2025-04-14 14:39:55,173 - cephci - ceph:1186 - DEBUG - Last metadata expiration check: 0:57:29 ago on Mon Apr 14 13:42:05 2025.
2025-04-14 14:39:55,405 - cephci - ceph:1186 - DEBUG - Package python3-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:55,406 - cephci - ceph:1186 - DEBUG - Package python3-pip-21.3.1-1.el9.noarch is already installed.
2025-04-14 14:39:55,407 - cephci - ceph:1186 - DEBUG - Package fio-3.35-1.el9.x86_64 is already installed.
2025-04-14 14:39:55,408 - cephci - ceph:1186 - DEBUG - Package fuse-2.9.9-16.el9.x86_64 is already installed.
2025-04-14 14:39:55,409 - cephci - ceph:1186 - DEBUG - Package ceph-fuse-2:19.2.1-126.el9cp.x86_64 is already installed.
2025-04-14 14:39:55,410 - cephci - ceph:1186 - DEBUG - Package attr-2.5.1-3.el9.x86_64 is already installed.
2025-04-14 14:39:55,411 - cephci - ceph:1186 - DEBUG - Package gcc-11.5.0-5.el9_5.x86_64 is already installed.
2025-04-14 14:39:55,412 - cephci - ceph:1186 - DEBUG - Package python3-devel-3.9.21-1.el9_5.x86_64 is already installed.
2025-04-14 14:39:55,413 - cephci - ceph:1186 - DEBUG - Package git-2.43.5-2.el9_5.x86_64 is already installed.
2025-04-14 14:39:55,415 - cephci - ceph:1186 - DEBUG - Package lua-5.4.4-4.el9.x86_64 is already installed.
2025-04-14 14:39:55,505 - cephci - ceph:1186 - DEBUG - Dependencies resolved.
2025-04-14 14:39:55,506 - cephci - ceph:1186 - DEBUG - Nothing to do.
2025-04-14 14:39:55,506 - cephci - ceph:1186 - DEBUG - Complete!
2025-04-14 14:39:56,651 - cephci - ceph:1606 - INFO - Execution of yum install -y --nogpgcheck python3 python3-pip fio fuse ceph-fuse attr gcc python3-devel git lua on 10.0.195.142 took 3.74234 seconds
2025-04-14 14:39:56,653 - cephci - ceph:1576 - INFO - Execute pip3 install xattr numpy scipy on 10.0.195.142
2025-04-14 14:39:57,658 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: xattr in /usr/local/lib64/python3.9/site-packages (1.1.4)
2025-04-14 14:39:57,658 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: numpy in /usr/local/lib64/python3.9/site-packages (2.0.2)
2025-04-14 14:39:57,658 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: scipy in /usr/local/lib64/python3.9/site-packages (1.13.1)
2025-04-14 14:39:57,658 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: cffi>=1.16.0 in /usr/local/lib64/python3.9/site-packages (from xattr) (1.17.1)
2025-04-14 14:39:57,659 - cephci - ceph:1186 - DEBUG - Requirement already satisfied: pycparser in /usr/local/lib/python3.9/site-packages (from cffi>=1.16.0->xattr) (2.22)
2025-04-14 14:39:57,659 - cephci - ceph:1186 - ERROR - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
2025-04-14 14:39:57,659 - cephci - ceph:1606 - INFO - Execution of pip3 install xattr numpy scipy on 10.0.195.142 took 1.005849 seconds
2025-04-14 14:39:57,661 - cephci - ceph:1576 - INFO - Execute ls /home/cephuser on 10.0.195.142
2025-04-14 14:39:58,664 - cephci - ceph:1606 - INFO - Execution of ls /home/cephuser on 10.0.195.142 took 1.003378 seconds
2025-04-14 14:39:58,666 - cephci - ceph:1576 - INFO - Execute rpm -qa | grep -w 'dbench' on 10.0.195.142
2025-04-14 14:39:59,669 - cephci - ceph:1606 - INFO - Execution of rpm -qa | grep -w 'dbench' on 10.0.195.142 took 1.00261 seconds
2025-04-14 14:39:59,671 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 14:40:00,675 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.003379 seconds
2025-04-14 14:40:00,675 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 14:40:00,677 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 14:40:01,681 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.003597 seconds
2025-04-14 14:40:01,682 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251
2025-04-14 14:40:02,686 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node8 on 10.0.195.251 took 1.003431 seconds
2025-04-14 14:40:02,688 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 14:40:03,692 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node8 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003688 seconds
2025-04-14 14:40:03,694 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251
2025-04-14 14:40:04,697 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node8.keyring on 10.0.195.251 took 1.003069 seconds
2025-04-14 14:40:04,699 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 14:40:04,700 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 14:40:05,704 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003244 seconds
2025-04-14 14:40:05,705 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3
2025-04-14 14:40:06,709 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node9 on 10.0.195.3 took 1.003222 seconds
2025-04-14 14:40:06,710 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 14:40:07,713 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node9 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.003088 seconds
2025-04-14 14:40:07,715 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3
2025-04-14 14:40:08,718 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node9.keyring on 10.0.195.3 took 1.003108 seconds
2025-04-14 14:40:08,719 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 14:40:08,720 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 14:40:09,724 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.003719 seconds
2025-04-14 14:40:09,726 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202
2025-04-14 14:40:10,729 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node10 on 10.0.195.202 took 1.003184 seconds
2025-04-14 14:40:10,731 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 14:40:11,734 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node10 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 1.003215 seconds
2025-04-14 14:40:11,736 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202
2025-04-14 14:40:12,740 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node10.keyring on 10.0.195.202 took 1.003534 seconds
2025-04-14 14:40:12,740 - cephci - cephfs_utilsV1:729 - INFO - Giving required permissions for clients:
2025-04-14 14:40:12,742 - cephci - ceph:1576 - INFO - Execute ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 14:40:13,746 - cephci - ceph:1606 - INFO - Execution of ceph auth get client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.003474 seconds
2025-04-14 14:40:13,747 - cephci - ceph:1576 - INFO - Execute ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142
2025-04-14 14:40:14,750 - cephci - ceph:1606 - INFO - Execution of ceph auth del client.ceph-regression-vhr823-futhjo-node11 on 10.0.195.142 took 1.003092 seconds
2025-04-14 14:40:14,752 - cephci - ceph:1576 - INFO - Execute ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 14:40:15,756 - cephci - ceph:1606 - INFO - Execution of ceph auth get-or-create client.ceph-regression-vhr823-futhjo-node11 mon 'allow *' mds 'allow *, allow * path=/' osd 'allow *' -o /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003491 seconds
2025-04-14 14:40:15,758 - cephci - ceph:1576 - INFO - Execute chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142
2025-04-14 14:40:16,761 - cephci - ceph:1606 - INFO - Execution of chmod 644 /etc/ceph/ceph.client.ceph-regression-vhr823-futhjo-node11.keyring on 10.0.195.142 took 1.003516 seconds
2025-04-14 14:40:16,765 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7,ceph-regression-vhr823-futhjo-node6 on 10.0.195.251
2025-04-14 14:40:17,769 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster create cephfs-nfs ceph-regression-vhr823-futhjo-node7,ceph-regression-vhr823-futhjo-node6 on 10.0.195.251 took 1.003404 seconds
2025-04-14 14:40:17,769 - cephci - cephfs_volume_management:23 - INFO - Wait for the process to start or stop
2025-04-14 14:40:17,771 - cephci - ceph:1576 - INFO - Execute ceph orch ps | grep cephfs-nfs on 10.0.195.251
2025-04-14 14:40:18,775 - cephci - ceph:1606 - INFO - Execution of ceph orch ps | grep cephfs-nfs on 10.0.195.251 took 1.003459 seconds
2025-04-14 14:40:23,779 - cephci - ceph:1576 - INFO - Execute ceph orch ps | grep cephfs-nfs on 10.0.195.251
2025-04-14 14:40:24,784 - cephci - ceph:1606 - INFO - Execution of ceph orch ps | grep cephfs-nfs on 10.0.195.251 took 1.003474 seconds
2025-04-14 14:40:24,786 - cephci - ceph:1576 - INFO - Execute ceph nfs cluster ls -f json on 10.0.195.251
2025-04-14 14:40:25,789 - cephci - ceph:1606 - INFO - Execution of ceph nfs cluster ls -f json on 10.0.195.251 took 1.003459 seconds
2025-04-14 14:40:25,790 - cephci - 2_node_nfs:88 - INFO - ceph nfs cluster created successfully
2025-04-14 14:40:25,791 - cephci - ceph:1576 - INFO - Execute ceph fs ls --format json-pretty on 10.0.195.251
2025-04-14 14:40:26,795 - cephci - ceph:1606 - INFO - Execution of ceph fs ls --format json-pretty on 10.0.195.251 took 1.003149 seconds
2025-04-14 14:40:26,796 - cephci - ceph:1576 - INFO - Execute ceph nfs export create cephfs cephfs-nfs /export_963 cephfs path=/ on 10.0.195.251
2025-04-14 14:40:28,801 - cephci - ceph:1606 - INFO - Execution of ceph nfs export create cephfs cephfs-nfs /export_963 cephfs path=/ on 10.0.195.251 took 2.004525 seconds
2025-04-14 14:40:28,803 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:40:29,807 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251 took 1.003502 seconds
2025-04-14 14:40:29,809 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:42:34,947 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251 took 125.138477 seconds
2025-04-14 14:42:34,948 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 60 seconds...
2025-04-14 14:43:34,994 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:43:35,999 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251 took 1.003735 seconds
2025-04-14 14:43:36,001 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:45:41,142 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251 took 125.140686 seconds
2025-04-14 14:45:41,143 - cephci - retry:28 - WARNING - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251, Retrying in 120 seconds...
2025-04-14 14:47:41,228 - cephci - ceph:1576 - INFO - Execute mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:47:42,233 - cephci - ceph:1606 - INFO - Execution of mkdir -p /mnt/nfs_ROHG2 on 10.0.195.251 took 1.004687 seconds
2025-04-14 14:47:42,234 - cephci - ceph:1576 - INFO - Execute mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:49:47,376 - cephci - ceph:1606 - INFO - Execution of mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 on 10.0.195.251 took 125.140937 seconds
2025-04-14 14:49:47,377 - cephci - 2_node_nfs:156 - INFO - mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251
2025-04-14 14:49:47,377 - cephci - 2_node_nfs:157 - INFO - Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/2_node_nfs.py", line 113, in run
    rc = fs_util.cephfs_nfs_mount(
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/utility/retry.py", line 32, in f_retry
    return f(*args, **kwargs)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_utilsV1.py", line 3277, in cephfs_nfs_mount
    client.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: mount -t nfs -o port=2049 ceph-regression-vhr823-futhjo-node7:/export_963 /mnt/nfs_ROHG2 returned mount.nfs: Connection refused
 and code 32 on 10.0.195.251

2025-04-14 14:49:47,378 - cephci - 2_node_nfs:160 - INFO - Cleaning up the system
2025-04-14 14:49:47,379 - cephci - ceph:1576 - INFO - Execute rm -rf /mnt/nfs_ROHG2/* on 10.0.195.251
2025-04-14 14:49:48,382 - cephci - ceph:1606 - INFO - Execution of rm -rf /mnt/nfs_ROHG2/* on 10.0.195.251 took 1.003079 seconds
2025-04-14 14:49:48,384 - cephci - ceph:1576 - INFO - Execute umount /mnt/nfs_ROHG2 on 10.0.195.251
2025-04-14 14:49:49,388 - cephci - ceph:1606 - INFO - Execution of umount /mnt/nfs_ROHG2 on 10.0.195.251 took 1.003726 seconds
2025-04-14 14:49:49,389 - cephci - run:854 - ERROR - umount /mnt/nfs_ROHG2 returned umount: /mnt/nfs_ROHG2: not mounted.
 and code 32 on 10.0.195.251
Traceback (most recent call last):
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/run.py", line 839, in run
    rc = test_mod.run(
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/2_node_nfs.py", line 168, in run
    client1.exec_command(sudo=True, cmd=command)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 2305, in exec_command
    return self.node.exec_command(cmd=cmd, **kw)
  File "/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/ceph/ceph.py", line 1700, in exec_command
    raise CommandFailed(
ceph.ceph.CommandFailed: umount /mnt/nfs_ROHG2 returned umount: /mnt/nfs_ROHG2: not mounted.
 and code 32 on 10.0.195.251
2025-04-14 14:49:49,391 - cephci - ceph:1576 - INFO - Execute podman --version | awk {'print $3'} on 10.0.195.21
2025-04-14 14:49:50,394 - cephci - ceph:1606 - INFO - Execution of podman --version | awk {'print $3'} on 10.0.195.21 took 1.003235 seconds
2025-04-14 14:49:50,395 - cephci - run:1065 - INFO - Podman Version 5.2.2
2025-04-14 14:49:50,396 - cephci - ceph:1576 - INFO - Execute docker --version | awk {'print $3'} on 10.0.195.21
2025-04-14 14:49:51,400 - cephci - ceph:1606 - INFO - Execution of docker --version | awk {'print $3'} on 10.0.195.21 took 1.003668 seconds
2025-04-14 14:49:51,402 - cephci - ceph:1576 - INFO - Execute ceph --version | awk '{print $3}' on 10.0.195.251
2025-04-14 14:49:52,406 - cephci - ceph:1606 - INFO - Execution of ceph --version | awk '{print $3}' on 10.0.195.251 took 1.003487 seconds
2025-04-14 14:49:52,406 - cephci - run:1081 - INFO - ceph Version 19.2.1-126.el9cp
2025-04-14 14:49:52,407 - cephci - run:1040 - INFO - ceph_clusters_file rerun/regression-vHr823-FUTHJO
2025-04-14 14:49:52,407 - cephci - run:927 - INFO - Test <module 'cephfs_nfs.2_node_nfs' from '/home/jenkins/ceph-builds/openstack/IBM/8.1/rhel-9/Regression/19.2.1-126/cephfs/99/cephci/tests/cephfs/cephfs_nfs/2_node_nfs.py'> failed
